modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -20,20 +20,19 @@ def index
         is_using_pagination = params[:page].present? || params[:per_page].present?
         resource.data = params[:include_image].present? ? fetch_and_include_images(resource.data) : resource.data
         resource = is_using_pagination ? resource.paginate(**pagination_params) : resource
-        render json: resource.data,
-               serializer: CollectionSerializer,
-               each_serializer: PrescriptionDetailsSerializer,
-               meta: resource.metadata
+
+        options = { meta: resource.metadata }
+        options[:links] = pagination_links(resource) if is_using_pagination
```

Comment:
@stevenjcumming Seeing an error for this when running locally:

![Screenshot 2024-07-29 at 2 52 42 PM](https://github.com/user-attachments/assets/3e922914-72ea-43f7-bb03-b3e798b8cd9f)

Reply:
It seems my concern didn't load. I'll require it on the controller 👍